contracts/modules/STO/USDTieredSTO.sol

```diff
@@ -319,16 +329,23 @@ contract USDTieredSTO is ISTO, ReentrancyGuard {
     function _modifyAddresses(
         address _wallet,
         address _reserveWallet,
-        address _usdToken
+        address[] _usdTokens
     ) internal {
         require(_wallet != address(0) && _reserveWallet != address(0), "Invalid wallet");
-        if (fundRaiseTypes[uint8(FundRaiseType.DAI)]) {
-            require(_usdToken != address(0), "Invalid usdToken");
-        }
         wallet = _wallet;
         reserveWallet = _reserveWallet;
-        usdToken = IERC20(_usdToken);
-        emit SetAddresses(_wallet, _reserveWallet, _usdToken);
+        _modifyUSDTokens(_usdTokens);
+    }
+
+    function _modifyUSDTokens(address[] _usdTokens) internal {
+        for(uint256 i = 0; i < usdTokens.length; i++) {
+            usdTokenEnabled[usdTokens[i]] = false;
+        }
+        usdTokens = _usdTokens;
+        for(i = 0; i < _usdTokens.length; i++) {
+            usdTokenEnabled[_usdTokens[i]] = true;
```

Comment:
can add the 0x0 check of an address of usdTokens

Reply:
Added.
Thanks @satyamakgec